content/using/install.md

```diff
@@ -91,43 +77,23 @@ http: live (insecure, loopback) on 12321
 ~sampel_marzod:dojo>
 ```
 
-When your ship is finished booting, you will see either the `~sampel_marzod:dojo>` or `~sampel_marzod:chat-cli/` prompt. If you see `:chat-cli`, press `Ctrl-X` to switch into Dojo.
+When your comet is finished booting, you will see either the `~sampel_marzod:dojo>` or `~sampel_marzod:chat-cli/` prompt. If you see `:chat-cli`, press `Ctrl-X` to switch into Dojo (the Urbit command line).
 
-To exit Urbit, use `Ctrl-D` or enter `|exit` into Dojo.
-
-To start your ship up again, run the following from your `urbit` directory (note the lack of `-c` argument):
+Get your password to log into landscape (the Urbit web interface) by typing:
 
 ```sh
-./urbit mycomet
-```
-
-### The Dojo
-
-Let's try our first command in the Dojo, the Urbit OS command line and Hoon REPL, to get oriented.
-
-Type `(add 2 2)` into the Dojo. You should see the following:
-
-```
-~sampel_marzod:dojo> (add 2 2)
++code
```

Comment:
and typing return, presumably

Reply:
We're not telling people to type return anywhere else.  Not sure this extra instruction is needed.